### ISSUE TITLE
Include the name of the asset's group in the asset listing command

### ISSUE THREAD

**Issue by benarmston** [alces-flight/flight-asset-cli#5]
Currently, the output of `list-assets` looks like:

```
+------------+-------------+--------------+
|Name        |Support Type |Decommissioned|
+------------+-------------+--------------+
|ben-asset-01|collaborative|false         |
|ben-asset-02|advice       |false         |
|ben-asset-03|advice       |false         |
|ben-asset-04|advice       |false         |
|ben-asset-05|managed      |false         |
+------------+-------------+--------------+
```

We should extend this to include the group name.  E.g.,

```
+------------+-------------+-------+--------------+
|Name        |Support Type |Group  |Decommissioned|
+------------+-------------+-------+--------------+
|ben-asset-01|collaborative|Group 1|false         |
|ben-asset-02|advice       |Group 2|false         |
|ben-asset-03|advice       |Group 1|false         |
|ben-asset-04|advice       |Group 2|false         |
|ben-asset-05|managed      |Group 3|false         |
+------------+-------------+-------+--------------+
```

We should keep an eye on the performance cost of doing so.  If costly, we might wish to make this opt in.

**Comment by WilliamMcCumstie** [comment]
WIP

**Comment by WilliamMcCumstie** [comment]
Implemented in version `0.3.0`. Will revisit if the performance is affected.
